src/lib/ngsi10/UpdateContextRequest.cpp

```diff
@@ -70,6 +75,28 @@ UpdateContextRequest::UpdateContextRequest(const std::string _contextProvider, E
 */
 void UpdateContextRequest::init(void)
 {
+  xmls  = 0;
+  jsons = 0;
+}
+
+
+
+/* ****************************************************************************
+*
+* UpdateContextRequest::format - 
+*/
+Format UpdateContextRequest::format(void)
+{
+  if (xmls > jsons)
```

Comment:
The numbers are increased where the attributes are 'sorted', right after receiving the responses for the forwarded messages.